pkg/apis/nfd/v1alpha1/annotations_labels.go

```diff
@@ -46,4 +46,10 @@ const (
 
 	// NodeTaintsAnnotation is the annotation that holds the taints that nfd-master set on the node
 	NodeTaintsAnnotation = AnnotationNs + "/taints"
+
+	// NodeFeatureObjNodeNameLabel is the label that specifies which node the
+	// NodeFeature object is targeting. Creators of NodeFeature objects must
+	// set this label and consumers of the objects are supposed to use the
+	// label for filterint features designated for a certain node.
+	NodeFeatureObjNodeNameLabel = "nfd.node.kubernetes.io/node-name"
```

Comment:
We do (in nfd-worker) but we don't care about the name really. Labels are the K8s way of filtering objects. Also, you might have two objects in the same namespace targeting the same node

Reply:
Sorry I was about naming NodeFeature based on the name of a node for the filtering purpose. Basically, filter based on the node name instead of labels. Although I can't imagine the reason for having more than one NodeFeature for a single node, filtering based on labels is reasonable for N>1 NodeFeature for a single node.

Also, there is a typo to be fixed :)

Reply:
> Sorry I was about naming NodeFeature based on the name of a node for the filtering purpose. Basically, filter based on the node name instead of labels. Although I can't imagine the reason for having more than one NodeFeature for a single node, filtering based on labels is reasonable for N>1 NodeFeature for a single node.

If/when NodeFeature objects are used for 3rd party extensions of nfd, there will be multiple objects per node. As said, there could even be multiple objects per node in the same namespace in which case name does not work anymore

> Also, there is a typo to be fixed :)

Yeah, thanks. I fixed that